### ISSUE TITLE
True, false values in $allDefPopQueries must not be escaped

### ISSUE THREAD

**Issue by Michiel-s** [AmpersandTarski/ampersand#148]
For
```
REPRESENT Boolean TYPE BOOLEAN
POPULATION Boolean CONTAINS ["no","yes"]
```
The population queries for the installer are escaped. This is not accepted as valid input by the database. Removing the escaped single quotes (\') solves the isse
```
INSERT INTO "Boolean"
              ("Boolean")
           VALUES (\'False\')
                , (\'True\')'
```

**Comment by Michiel-s** [comment]
Hi Han,

Is this issue fixed with merge a501df66e59d36ffa147b284b3c183ac032b1975?

**Comment by hanjoosten** [comment]
Yes, it is.
